libshaderc_spvc/src/spvc_private.cc

```diff
@@ -65,15 +66,34 @@ void consume_spirv_tools_message(shaderc_spvc_context* context,
                                  spv_message_level_t level, const char* src,
                                  const spv_position_t& pos,
                                  const char* message) {
-  context->messages.push_back(message);
+  switch (level) {
+    case SPV_MSG_FATAL:
+      shaderc_spvc::ErrorLog(context) << "spirv-tools: " << message;
```

Comment:
This is changing the previous behavour for error messages by prepending spirv-tools

Reply:
Done